packages/design/components/Message/style/index.less

```
@@ -0,0 +1,36 @@
+@import '../../../theme/base';
+
+.bgm-message {
+  display: inline-block;
+  width: auto;
+
+  &-full {
+    width: 100%;
+  }
+
+  > div {
+    border-radius: 19px;
+    font-weight: 600;
+    font-size: 14px;
+    line-height: 20px;
+    padding: 9px 16px;
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+
+    > svg {
+      flex: none;
+      margin-right: 8px;
+    }
+  }
+
+  &-error > div {
```

```suggestion
  &--error > div {
```
这应该属于修饰符了
https://github.com/bangumi/frontend/blob/master/docs/css-naming-convention.md